src/saveload/saveload.h

```diff
@@ -371,6 +371,7 @@ enum SaveLoadVersion : uint16_t {
 	SLV_ECONOMY_DATE,                       ///< 326  PR#10700 Split calendar and economy timers and dates.
 	SLV_ECONOMY_MODE_TIMEKEEPING_UNITS,     ///< 327  PR#11341 Mode to display economy measurements in wallclock units.
 	SLV_CALENDAR_SUB_DATE_FRACT,            ///< 328  PR#11428 Add sub_date_fract to measure calendar days.
+	SLV_INFINITE_MONEY,                     ///< 329  PR#11902 Add infinite money mode
```

Comment:
I don't think adding a new setting requires a savegame bump. There are no conversions.

Reply:
And adding of settings no longer needs a savegame bump either. It used to, but we solved that a few version ago.